files/SZWESAMP/ZWEGENER

```diff
@@ -0,0 +1,89 @@
+//ZWEGENER JOB                                                                  
+//*                                                                             
+//* This job is responsible for generating other jobs required                  
+//* to configure Zowe.                                                          
+//*                                                                             
+//* The method of validating your configuration is using
+//* JSON Schema <https://json-schema.org>. Zowe provides
+//* the ConfigMgr to assist in this. This job will invoke
+//* the ConfigMgr to validate your current configuration
+//* before generating any jobs. If there are any values
+//* that are incorrect, you will be notified. You should
+//* fix the value and then run this job again. You can run
+//* this job as many times as you need.
+//*
+//* Configmgr documentation:
+//* https://docs.zowe.org/stable/user-guide/configmgr-using
+//*
+//* Note: Any string with braces has an associated yaml value
+//*       in one of the yaml definitions for Zowe.
+//*       You must find the value and substitute it.
+//*
+//*       {key} -> value
+//*
+//GENER    EXEC PGM=IKJEFT1B
+//ISPPROF  DD   DSN=,DISP=(NEW,DELETE),UNIT=,
+//   DCB=(RECFM=FB,LRECL=80,BLKSIZE=3120,DSORG=PO),
+//   SPACE=(3120,(20,5,10))
+//*
+//* Replace {zowe.setup.dataset.prefix} with the
+//* Value as seen in zowe.yaml
+//*
+//SYSPROC  DD   DSN={zowe.setup.dataset.prefix}.SZWEEXEC,DISP=SHR
+//*
+//* Replace {zowe.setup.dataset.prefix} with the
+//* Value as seen in zowe.yaml
+//*
+//STEPLIB  DD   DSN={zowe.setup.dataset.prefix}.SZWELOAD,DISP=SHR
+//ISPPLIB  DD   DSN=ISP.SISPPENU,DISP=SHR
+//ISPMLIB  DD   DSN=ISP.SISPMENU,DISP=SHR
+//ISPTLIB  DD   DSN=ISP.SISPTENU,DISP=SHR
+//ISPSLIB  DD   DSN=ISP.SISPSENU,DISP=SHR
+//*
+//* The order must be as follows.
+//*
+//* zowe-yaml-schema.json
+//* server-common.json
+//*
+//* Replace {zowe.runtimeDirectory} with where your Zowe run time
+//* directory is, as seen in zowe.yaml
+//*
+//MYSCHEMA DD   *,DLM=$$
+FILE {zowe.runtimeDirectory}/schemas/zowe-yaml-schema.json
```

Comment:
How it is with string continuation here?
```
//MYSCHEMA DD   *,DLM=$$
----+----1----+----2----+----3----+----4----+----5----+----6----+----7----+----8                                                        
FILE /zowe/---1----+----2----+----3----+----4----+/schemas/zowe-yaml-schema.json
FILE {zowe.runtimeDirectory}/schemas/server-common.json                         
```
The `zowe.runtimeDirectory` must fit into 45 chars, which is fine for normal users, but not for me...

Special chars are not allowed. When space is in the zowe dir name, backslash or quotes will not help.

Reply:
Are we able to do SET statements such as?

SET RUNTIME=
SET COMMONJSON=/schemas/server-common.json

FILE &RUNTIME.&COMMONJSON